ztp/ztp-ran-policy-generator/sourcePolicies/SriovSubscription.yaml

```diff
@@ -0,0 +1,11 @@
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: sriov-network-operator-subscription
+  namespace: openshift-sriov-network-operator
+spec:
+  channel: "4.7"
```

Comment:
4.8 ?

Reply:
it can be updated in the ranGen template but I mentioned that we do recommend update the channel with repo/branch not just update the file.

Reply:
Agreed. We should start with 4.8 as a baseline